src/LibraryManager/Providers/Unpkg/UnpkgProvider.cs

```diff
@@ -39,7 +39,7 @@ public override string GetSuggestedDestination(ILibrary library)
         {
             if (library != null && library is UnpkgLibrary unpkgLibrary)
             {
-                return unpkgLibrary.Name;
+                return unpkgLibrary.Name.TrimStart('@');
```

Comment:
>Name [](start = 36, length = 4)

Nit: Ever possible for the name to be null? I assume not, otherwise we could do ?.

Reply:
It is technically possible (though I don't think it would happen currently), but better safe than sorry. 👍